src/diceBot/ParanoiaRebooted.rb

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
+# frozen_string_literal: true
+
+class ParanoiaRebooted < DiceBot
+  # ゲームシステムの識別子
+  ID = 'ParanoiaRebooted'
+
+  # ゲームシステム名
+  NAME = 'パラノイア リブーテッド'
+
+  # ゲームシステム名の読みがな
+  SORT_KEY = 'ぱらのいありぶーてっど'
```

Comment:
```suggestion
  SORT_KEY = 'はらのいありふうてつと'
```
SORT_KEYの設定方法は若干特殊です。
詳しくは下記を参照してください。
https://github.com/bcdice/BCDice/blob/master/docs/dicebot_sort_key.md